cypher-shell/src/test/java/org/neo4j/shell/cli/CliArgHelperTest.java

```diff
@@ -150,9 +150,19 @@ public void parseFullAddress() {
         assertNotNull(cliArgs);
         assertEquals("alice", cliArgs.getUsername());
         assertEquals("foo", cliArgs.getPassword());
+        assertEquals("bolt+routing://", cliArgs.getScheme());
         assertEquals("bar", cliArgs.getHost());
         assertEquals(69, cliArgs.getPort());
     }
+    @Test
```

Comment:
should there be a blank line above this?

Reply:
Yes.